spec/controllers_spec.rb

```diff
@@ -121,6 +121,52 @@ def custom_timestamp_callback
       expect(response.body).to be_present
     end
 
+    context 'with random honeypot' do
+      context 'auto-scoped' do
+        it 'passes with no spam' do
+          post :categorize, params: { topic: { title: 'foo' } }
+
+          expect(response.body).to be_present
+        end
+
+        it 'fails with spam' do
+          post :categorize, params: { topic: { "#{InvisibleCaptcha.honeypots.sample}": 'foo' } }
+
+          expect(response.body).to be_blank
+        end
+      end
+
+      context 'with no scope' do
+        it 'passes with no spam' do
+          $DOPRY = true
```

Comment:
What's this `$DOPRY` var?

Reply:
Oops! That's my bad. Debug stuff I must've forgotten to remove.

Reply:
Ripped it out 🙂